README.md

```diff
@@ -110,9 +111,7 @@ wavefront      true          Running(1/1)        Running (1/1)        Running (3
 
 # Configuration
 
-The Wavefront Operator is deployed as a Custom Resource Definition and managing process,
-which allows the user to custom configure and `kubectl apply -f` a Custom Resource yaml file
-to the cluster to create an instance of the Operator. See below for configuration options.
+The Wavefront Operator is configured via a custom resource. When the resource is updated, the managing process (the operator) will pick up the changes and update the integration deployment accordingly. To update the custom resource, change the option you want in the the wavefront custom resource file and run `kubectl apply -f <your config file>.yaml`. See below for configuration options.
 
 ## Default option
```

Comment:
These options need more explanation. As a user, what would I be looking to accomplish that would make me decide to use one path other another?

Reply:
This should more accurately be on all of the configuration option sections below this marker to help the user know how to choose their adventure of which option to pursue.

Reply:
We believe we have resolved this by providing scenario/task based examples instead.